integration/spark2/src/main/scala/org/apache/spark/util/AlterTableUtil.scala

```diff
@@ -511,6 +515,21 @@ object AlterTableUtil {
     }
   }
 
+  def validateRangeColumnProperties(carbonTable: CarbonTable,
+      propertiesMap: mutable.Map[String, String]): Unit = {
+    if (propertiesMap.get(CarbonCommonConstants.RANGE_COLUMN).isDefined) {
+      val rangeColumnProp = propertiesMap.get(CarbonCommonConstants.RANGE_COLUMN).get
```

Comment:
As I mentioned earlier, we need to validate the number of fields here.

Reply:
ok